prow/cmd/branchprotector/protect_test.go

```diff
@@ -82,113 +83,6 @@ func TestOptions_Validate(t *testing.T) {
 	}
 }
 
-func TestJobRequirements(t *testing.T) {
```

Comment:
I think it is good to keep these tests, which will give us more confidence the refactor does not regress current behavior.

Reply:
I just moved them to prow/config/branch_protection_test.go